lib/pliny/commands/generator/schema.rb

```diff
@@ -38,34 +46,38 @@ def warn_legacy
         @warned_legacy = true
       end
 
+      def schema_path
+        if legacy?
+          "./docs/schema"
+        else
+          "./schema"
+        end
+      end
+
       def schema_json_path
         if legacy?
```

Comment:
No, because one needs to be `./docs/schema.json`, and the other one `./schema/schema.json`.
We need the conditional to handle the file being inside the schema folder or not.